lib/background_worker/logging_concern.rb

```diff
@@ -0,0 +1,17 @@
+require 'active_support/concern'
+
+module BackgroundWorker
+  module LoggingConcern
```

Comment:
I don't really like "Concern" in the class name; lets just copy AJ and call this "Logging"
https://github.com/rails/rails/blob/main/activejob/lib/active_job/logging.rb